### ISSUE TITLE
False positive: grindr.mobi

### ISSUE THREAD

**Issue by yokoffing** [badmojr/1Hosts#628]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

**Lists in use:**
- [ ] mini
- [ ] Lite
- [ ] Pro
- [x] Xtra
<br>

**Client:** NextDNS

<br>

**Domains:**
```
grindr.mobi
```
<br>

**Details:**
Completely breaks [Grindr app](https://apps.apple.com/us/app/grindr-gay-dating-chat/id319881193):

![5A894299-AF5C-4F5B-86F9-11576D4AC555](https://user-images.githubusercontent.com/11689349/181927101-561a6e14-afbd-4056-b56a-476f21b63370.png)

At the time of opening this issue, the only other request we block in `Xtra` vs. `Pro` is `presence.grindr.com`, which doesn't break the app, but it could interfere with detecting the distance of users from each other or displaying users online status. I'm not sure; I'll have to be on the lookout for breakage.

TL;DR: We can leave `presence.grindr.com` blocked in Xtra (for now, at least), but definitely should unblock `grindr.mobi` in all lists.

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>
